cardano-config/src/Cardano/Config/Types.hs

```diff
@@ -156,9 +156,12 @@ instance FromJSON GenesisFile where
   parseJSON invalid = panic $ "Parsing of GenesisFile failed due to type mismatch. "
                            <> "Encountered: " <> (T.pack $ show invalid)
 
-newtype DelegationCertFile = DelegationCertFile
-  { unDelegationCert :: FilePath }
+-- Encompasses staking certificates, stake pool certificates,
+-- genesis delegate certificates and MIR certificates.
+newtype CertificateFile = CertificateFile
+  { unCertificate :: FilePath }
   deriving newtype Show
+  deriving Eq
```

Comment:
Should be able to do:

```suggestion
  deriving newtype (Eq, Show)
```

That is, unless there's some reason for not doing that, that I'm not aware of.